subgraphs/erc20/schema.graphql

```diff
@@ -0,0 +1,293 @@
+# Subgraph Schema: ERC20 Tokens
+# Version: 1.0.0
+
+#############################
+####### Token Metadata ######
+#############################
+
+type Token @entity {
+  " Smart contract address of the token "
+  id: ID!
+
+  " Name of the token, mirrored from the smart contract "
+  name: String!
+
+  " Symbol of the token, mirrored from the smart contract "
+  symbol: String!
+
+  " The number of decimal places this token uses, default to 18 "
+  decimals: Int!
+
+  " If token transfers are paused, derived from paused event "
+  isPaused: Boolean
+
+  " Token Holders Balance "
+  holdersBalance: [AccountBalance!]! @derivedFrom(field: "token")
+
+  " Number of token holders "
+  holderCount: BigInt!
+
+  " Total number of token transfer transactions "

```

Comment:
probably want to be specific here about whether we are talking about events or transactions. If 1 tx contains 10 token transfers, I think we should consider it as 10 transfers (i.e. transferCount += 10)